### DISCUSSION TITLE
A conflict with shape key?

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#216] [Feedback & Feature Requests]
[conflict with shape key.zip](https://github.com/bnpr/Malt/files/7797949/conflict.with.shape.key.zip)

Here I found a bug that crashes blender.
Select the cube in the file, select any frame (such as frame 100), drag the shapekey value, then blender crashes.

**Comment by HSDSZ** [comment]
If blender does not crashes, auto keyframe another frame with a different shapekey value, blender will crashes

I am using blender 3.0 and a malt developing version

**Comment by pragma37** [comment]
I can't reproduce the issue, it works fine on my end.
Could you create a new issue with the Malt log and the Blender crash log?
https://github.com/bnpr/Malt/issues/new?assignees=&labels=&template=bug_report.yml

**Reply by HSDSZ** [reply]
Ok， new issue created
